muikku-atests/src/test/java/fi/otavanopisto/muikku/TestUtilities.java

```diff
@@ -1,8 +1,11 @@
 package fi.otavanopisto.muikku;
 
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import org.apache.commons.collections.map.HashedMap;
```

Comment:
Unused import (Ctrl+Shift+O when editing the source file gets rid of these easily)